pkg/controller/appdefinition/status.go

```diff
@@ -369,13 +375,34 @@ func podName(pod *corev1.Pod) string {
 	return pod.Labels[labels.AcornContainerName]
 }
 
-func endpoints(cfg *apiv1.Config, app *v1.AppInstance) string {
+func endpoints(req router.Request, cfg *apiv1.Config, app *v1.AppInstance) (string, error) {
 	endpointTarget := map[string][]v1.Endpoint{}
 	for _, endpoint := range app.Status.Endpoints {
 		target := fmt.Sprintf("%s:%d", endpoint.Target, endpoint.TargetPort)
 		endpointTarget[target] = append(endpointTarget[target], endpoint)
 	}
 
+	ingresses := &networkingv1.IngressList{}
+	err := req.List(ingresses, &kclient.ListOptions{
+		Namespace: app.Status.Namespace,
+		LabelSelector: klabels.SelectorFromSet(map[string]string{
+			labels.AcornManaged: "true",
+			labels.AcornAppName: app.Name,
+		}),
+	})
+	if err != nil {
+		return "", err
+	}
+
+	ingressTLSHosts := []string{}
```

Comment:
nitpick, a map would be more efficent. If this where a `map[string]interface{}`, line 423 would be a much more efficient lookup.

Reply:
Yup :)